adls-plugins/src/main/java/co/cask/hydrator/plugin/sink/ADLSBatchSink.java

```diff
@@ -128,10 +145,19 @@ public void transform(StructuredRecord input, Emitter<KeyValue<AvroKey<GenericRe
     @Macro
     public String fileSystemProperties;
 
-    @Description("Output schema for the output files.")
+    @Nullable
+    @Description("Output schema for avro format output files.")
     @Macro
     public String schema;
 
+    @Description("The format of output files.")
+    @Macro
+    private String outputFormat;
+
+    @Nullable
+    @Description("Field delimiter for text format output files.")
```

Comment:
should macro enable this field as well.

Reply:
you should have a default for this like a tab character. To set a default, set the value in a no-argument constructor. The description should mention the default value.